src/main/java/de/uniba/dsg/serverless/model/PerformanceData.java

```diff
@@ -30,7 +30,7 @@ public PerformanceData() {
 		this.memoryUsed = -1;
 	}
 
-	public PerformanceData(String functionName, String logStream, String requestId, long startTime, 
+	public PerformanceData(String functionName, String logStream, String requestId, long startTime,
```

Comment:
Two constructors are not necessary here. Please refactor this constructor and change the constructor call in AWSLogAnalyzer line 232. LocalDateTime startTime is the better option in this case.